src/components/StyleGuide/ColorCard.tsx

```diff
@@ -166,8 +275,12 @@ export const ColorCard = (props: React.PropsWithChildren<ColorCardProps>) => {
           )}
```

Comment:
I like `Base` because it points to the color it's based on. `Source` makes sense in the Figma file because that's the source you should go to to get the color. Maybe it's a design/dev thing? Let's move it to `Source` so it can be consistent with Figma.